src/engine/Simulator.cpp

```diff
@@ -19,6 +19,10 @@
 #include "MarabouError.h"
 #include "Simulator.h"
 
+#ifdef _WIN32
+#include "time.h"
```

Comment:
Is time.h a system file? If so, can we change to <time.h>?

Reply:
Yes this is a system file. Fixed with 7c28456.